docs/pages/index.md

```diff
@@ -32,7 +32,7 @@ To create a new documentation site, run `npm init docs` and follow the prompts.
 Once the application has been generated, see the [README.md](https://github.com/jxnblk/mdx-docs/create-docs/templates/next/README.md)
 for more documentation.
 
-To add MDX Docs to an existing Next.js app, see the [Custom Setup](custom-setup) docs.
+To add MDX Docs to an existing Next.js app, see the [Custom Setup](https://jxnblk.com/mdx-docs/custom-setup) docs.
```

Comment:
Thanks! This could probably be `/custom-setup` – if it doesn't work when built, I can dig in to make sure it does